src/arraydist.jl

```diff
@@ -18,9 +18,7 @@ function arraydist(dists::AbstractMatrix{<:UnivariateDistribution})
     return MatrixOfUnivariate(dists)
 end
 function Distributions._logpdf(dist::MatrixOfUnivariate, x::AbstractMatrix{<:Real})
-    # return sum(((d, xi),) -> logpdf(d, xi), zip(dist.dists, x))
-    # Broadcasting here breaks Tracker for some reason
-    return sum(map(logpdf, dist.dists, x))
+    return mapreduce((disti, xi) -> logpdf(disti, xi), +, dist.dists, x)
```

Comment:
Note that `mapreduce` does not check that `dist.dists` and `x` are of the same size. But that should be fine since checks should happen before `_logpdf` is called.
```suggestion
    return mapreduce(logpdf, +, dist.dists, x)
```